skfda/representation/_functional_data.py

```diff
@@ -822,7 +822,7 @@ def sum(  # noqa: WPS125
     def mean(
         self: T,
         *,
-        axis: None = None,
+        axis: Optional[int] = None,
```

Comment:
Use `int | None` instead.

Reply:
I used the other notation to be coherent with the rest of the file. 
Changed now.